addons/sms/tests/test_sms_template_access_rights.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests.common import SavepointCase, new_test_user
+from odoo.exceptions import AccessError
+
+
+class TestSmsTemplateAccessRights(SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.sms_manager = new_test_user(cls.env, login='sms_manager', groups='base.group_system')
+        cls.sms_user = new_test_user(cls.env, login='sms_user', groups='base.group_user')
+
+    def create_sms_template(self, user, model):
+        return self.env['sms.template'].with_user(user).create(dict({
```

Comment:
`dict()` is not necessary.

Please refactor to create on the list of values.
Creating in a loop is a bad practice (kills performances).

Reply:
done